homeassistant/components/ezviz/strings.json

```diff
@@ -58,5 +58,11 @@
         }
       }
     }
+  },
+  "issues": {
+    "manual_migration": {
+      "title": "Ezviz Detection sensitivity service is being removed",
+      "description": "Ezviz Detection sensitivity service is deprecated and will be removed in Home Assistant 2023.8; Please use Detection sensitivity entity to set."
```

Comment:
This is missing the instruction on how to "fix" / get rid of the issue. E.g., in this case it is probably something like: adjust the automation or script that uses the service and select submit below to mark this issue as resolved... or something like that.

Reply:
Done.